subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/ShowToolchainsTaskAction.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.jvm.toolchain.internal;
+
+import org.gradle.api.Action;
+import org.gradle.api.plugins.HelpTasksPlugin;
+
+public class ShowToolchainsTaskAction implements Action<ShowToolchainsTask> {
+
+    private final String projectName;
+
+    public ShowToolchainsTaskAction(String projectName) {
+        this.projectName = projectName;
+    }
+
+    @Override
+    public void execute(ShowToolchainsTask task) {
+        task.setDescription("Displays the java toolchains available for " + projectName + ".");
+        task.setGroup(HelpTasksPlugin.HELP_GROUP);
+        task.setImpliesSubProjects(true);
```

Comment:
:o: Does it?

Reply:
As discussed, it now indeed works as we auto-apply the plugin to the root project. Added a multi project integration test to cover that as well